doc/2/api/essentials/query-syntax/index.md

```diff
@@ -41,6 +41,27 @@ If the form field holds a file, then the corresponding JSON key will refer to an
 - `mimetype`: MIME type
 - `file`: file content, encoded in base64
 
+### JSON query endpoint
+
+Kuzzle also exposes an endpoint to send queries using the standard JSON request format used by other protocols.  
+
+This makes it possible to avoid with the use of HTTP routes and to send queries via the HTTP protocol in the same way as for other protocols.
```

Comment:
```suggestion
This makes it possible to avoid the use of the REST API and to send queries via the HTTP protocol the same way as for any other protocols.
```